mw-webapp/src/component/editableTextarea/editableTextarea.tsx

```diff
@@ -71,18 +74,22 @@ export const EditableTextarea = (props: EditableTextareaProps) => {
     }
   };
 
+  const handleChange = (event: ChangeEvent<HTMLTextAreaElement>) => {
+    const newText = event.target.value;
+    setText(newText);
+
+    if (event.target.parentNode instanceof HTMLElement) {
+      event.target.parentNode.dataset.replicatedValue = newText;
+    }
+  };
+
   /**
    * Render Textarea
    */
   const renderTextarea = () => (
-    <Textarea
-      defaultValue={text}
-      onChange={setText}
-      placeholder={props.placeholder ?? ""}
-      rows={props.rows}
-      isAutofocus
-      onKeyPress={handleCtrlEnter}
-    />
```

Comment:
Why did you deleted it?

Reply:
For some reason the built-in text field didn't want to become dynamic and I couldn't figure it out